models/data.js

```diff
@@ -47,81 +44,80 @@ function reader_norm(out, path, callback) {
     } catch (e) {
       console.error(
         "Exception while parsing file",
-        path + file,
+        pathname + file,
         ": ",
         e.message
       );
       return;
     }
-
     // TODO: There should be a try/catch around this
```

Comment:
didn't you take care of this TODO? can you please remove these comments if they've been addressed?